frontend/apps/web/app/(mgmt)/[account]/new/job/job-form-validations.ts

```diff
@@ -129,12 +140,14 @@ export type DefineFormValues = Yup.InferType<typeof DefineFormValues>;
 
 export const ConnectFormValues = SourceFormValues.concat(
   Yup.object({
-    destinations: Yup.array(NewDestinationFormValues).required(),
+    destinations: Yup.array(NewDestinationFormValues).required(
+      'At least one destination Connection is required'
```

Comment:
fwiw the `required` here simply says that the `destinations` property is required, not that it has a minimum length (we arent really concerned with that here). In this case it just updates the type to remove the undefined. 

This is generally true for most of the `required` functions (meaning: it's mostly for us to have nice types as we typically have defaults for everything due to how that interacts with react forms)

Reply:
yeah i was kinda going overkill to just set it